store/dataStore.test.ts

```diff
@@ -2,8 +2,19 @@ import { renderHook, act } from '@testing-library/react';
 import { useDataStore } from './dataStore';
 
 // test values
-const userId = '123';
-const userEmail = 'test@email.com';
+const userData = {
+   userId: '123',
+   userEmail: 'test@email.com',
+}
+
+const gameData = { 
+teamName: "Philadelphia Eagles",
```

Comment:
Is your prettier not working in save? It should've formatted these props.

Reply:
It wasn't. I'll update it and will look into as to why Prettier wasn't working.